src/app/loadout-builder/process/useProcess.ts

```diff
@@ -80,8 +80,8 @@ export function useProcess({
   subclass: ResolvedLoadoutItem | undefined;
   modStatChanges: ModStatChanges;
   armorEnergyRules: ArmorEnergyRules;
-  statOrder: number[];
-  statFilters: StatFilters;
+  statOrder: ArmorStatHashes[];
```

Comment:
`statOrder` turns out to still be useful in a lot of places because as I mentioned, `statConstraints` doesn't retain the ignored stats and thus can't remember their order (and it's just generally more annoying to use).